recipes/gromacs/build.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+
+# gromacs > 2021 requires a non default OSX version
+if [ "$(uname)" = 'Darwin' ] ; then
+    export MACOSX_DEPLOYMENT_TARGET=10.13 
+fi
+
+
+mkdir build
+cd build
+
+## See INSTALL of gromacs distro
+for ARCH in SSE2 AVX_256 AVX2_256; do
+  cmake_args=(
+    -DSHARED_LIBS_DEFAULT=ON
+    -DBUILD_SHARED_LIBS=ON
+    -DGMX_PREFER_STATIC_LIBS=NO
+    -DGMX_BUILD_OWN_FFTW=OFF
+    -DGMX_DEFAULT_SUFFIX=ON
+    # Tests are not currently run, so do not download them
+    # -DREGRESSIONTEST_DOWNLOAD=ON
+    -DCMAKE_PREFIX_PATH="${PREFIX}"
+    -DGMX_INSTALL_PREFIX="${PREFIX}"
+    -DCMAKE_INSTALL_PREFIX="${PREFIX}"
+    -DGMX_SIMD="${ARCH}"
+    -DCMAKE_INSTALL_BINDIR="bin.${ARCH}"
+    -DCMAKE_INSTALL_LIBDIR="lib.${ARCH}"
+    -DGMX_VERSION_STRING_OF_FORK="conda-forge"
+  )
+  # OpenCL header on Mac is not recognized by GROMACS
+  if [ "$(uname)" != 'Darwin' ] ; then
+      cmake_args+=(-DGMX_GPU=OpenCL)
+  fi
+  if [[ "${mpi}" == "nompi" ]]; then
+      cmake_args+=(-DGMX_MPI=OFF)
```

Comment:
```suggestion
      cmake_args+=(-DGMX_MPI=OFF -DGMX_THREAD_MPI=ON)
```
So one at least gets thread-mpi version of gromacs.